app/actions/index.js

```diff
@@ -23,6 +25,10 @@ window.fetch = new Fetch({
   binaryContentTypes: ["image/", "video/", "audio/", "application/gzip"]
 }).build();
 
+const obsdb = createOsmp2p("obs");
+const osmdb = createOsmp2p("osm");
+const osm = osmp2p(obsdb, osmdb);
```

Comment:
I still think there might be race conditions for Level operations, but admittedly I'm not really familiar with how AsyncStorage works underneath. Re: removing the other DB instance though: :+1:!